Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -292,6 +291,12 @@ function Set-TestAgentConfiguration
         InvokeDTAExecHostExe -Version $TestAgentVersion -MachineCredential $MachineUserCredential | Out-Null
     }
 
+    if ($configAsProcess -eq $true)
+    {  
+        Write-Verbose -Message "Trying to configure power options so that the console session stays active" -Verbose
+        ConfigurePowerOptions -MachineCredential $MachineUserCredential
```

Comment:
Use Out-Null , so that the output of this command doesn't impact the outcome of the Set-TestAgentConfiguration